CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Change Log
 
+## 1.14.0 (May 29, 2024)
+
+ * [#4101](https://github.com/redhat-developer/vscode-openshift-tools/issues/4101) Project not listed if it has a display name annotation.
+ * [#4092](https://github.com/redhat-developer/vscode-openshift-tools/issues/4092) Trying to create a serverless function using the Spring Boot template causes the webview to fail.
+ * [#4086](https://github.com/redhat-developer/vscode-openshift-tools/issues/4086) Create an web view for supporting to create routes from the extension .
+ * [#4083](https://github.com/redhat-developer/vscode-openshift-tools/issues/4083) Helm chart browser broken.
```

Comment:
Fix Helm chart browser sometimes showing no charts

Reply:
@rgrunber has suggested "Helm charts may not load on Linux under certain conditions." for this one